test/ingestors/fxhash.test.ts

```diff
@@ -0,0 +1,49 @@
+import { expect } from 'chai';
+import { FxHashIngestor } from '../../src/ingestors/fxhash';
+import { mintIngestorResources } from '../resources';
+import { EVMMintInstructions } from '../../src/lib/types/mint-template';
+import { MintTemplateBuilder } from '../../src/lib/builder/mint-template-builder';
+
+describe('fxhash', function () {
+  it('supportsUrl: Returns false for an unsupported URL', async function () {
+    const ingestor = new FxHashIngestor();
+    const url = 'https://example.com';
+    const result = await ingestor.supportsUrl(url);
+    expect(result).to.be.false;
+  });
+  it('supportsUrl: Returns true for a supported URL', async function () {
+    const ingestor = new FxHashIngestor();
+    const url = 'https://www.fxhash.xyz/generative/slug/1x4-shape-study';
+    const result = await ingestor.supportsUrl(url);
+    expect(result).to.be.true;
+
+    const url2 = 'https://fxhash.xyz/generative/slug/allegro';
+    const result2 = await ingestor.supportsUrl(url2);
+    expect(result2).to.be.true;
+  });
+  it('createMintTemplateForUrl: Returns a mint template for a supported URL', async function () {
```

Comment:
Would be great to see a test for each intended supported chain.

Not least because... I suspect that it may only support Base use of Alchemy right now. 

I'll run these through our internal ingestion later today too and make a couple updates to this repo to make testing easier.

🥳 🥳 🥳 🥳 🥳 🥳

Reply:
Ha! Good catch, adding now :)